.github/workflows/chromatic.yaml

```diff
@@ -25,6 +25,7 @@ jobs:
               - 'packages/**'
               - 'draft-packages/**'
               - 'storybook/**'
+              - 'docs/**'
```

Comment:
You'll only want docs that are stories since there are `.md` files in this directory which will never affect Chromatic, so you'll want to check for `.stories.tsx` or `stories.mdx` only.
I'm guessing it will be something like this `'docs/**/*.stories.(mdx|tsx)'` but I'm not 100% sure (maybe it would need to be separate lines per extension), so you'll want to check using `run chromatic` :)